packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -328,7 +332,7 @@ export class ProductAttributeValue extends PureComponent {
         const isSwatch = label;
 
         if (isFormattedAsText) {
-            return label || value || __('N/A');
+            return label || <Html content={ `${value}` } /> || __('N/A');
```

Comment:
`<Html />` component will always give `true` here, so `_('N/A')` will never be rendered.
But what if the value is empty here?